app/services/exports/export_adjustments_csv_service.rb

```diff
@@ -0,0 +1,52 @@
+module Exports
+  class ExportAdjustmentsCSVService
+    def initialize(adjustments:, organization:)
```

Comment:
I generally prefer module-level methods to instance level. They're easier to stub and work with. So e.g.

```ruby
module Exports
  module ExportAdjustmentsCSVService
    class << self
      def generate_csv(adjustments, organization)
        CSV.generate(headers: true) do |csv|
          generate_csv_data(adjustments, organization).each { |row| csv << row }
        end
      end
    end
  end
end
```

Reply:
Oh this would be absolutely better as module methods, thanks